db/seeds.rb

```diff
@@ -1,8 +1,28 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+require 'seed_methods'
 
 class AddInitialObjects < ActiveRecord::Migration
 
+  rando_pass = Devise.friendly_token.first(8)
+  accounts = {'manydeposits@example.com' => 'Many Deposits', 'nodeposits@example.com' => 'No Deposits', 'manager@example.com' => 'Repository Manager', 'delegate@example.com' => 'Student Delegate'} 
```

Comment:
Let's remove the repository manager account from this hash since we have a new, separate admin account that fills this function.

accounts = {'manydeposits@example.com' => 'Many Deposits', 'nodeposits@example.com' => 'No Deposits', ~~'manager@example.com' => 'Repository Manager',~~ 'delegate@example.com' => 'Student Delegate'}